src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -688,6 +688,10 @@
 			<field name="FirstChanceException"/>
 		</type>
 
+		<!-- mono_method_has_unmanaged_callers_only_attribute () -->
+		<!-- aot-compiler.c -->
```

Comment:
the method is in marshal.c

Reply:
I looked at the method, is the issue that `mono_class_try_get_unmanaged_callers_only_attribute_class` not gracefully handles the case where the attribute was linked out?

Reply:
The runtime/aot compiler needs this attribute, and the linker removes it from methods when using --used-attrs-only.

Reply:
The PInvokeTableGenerator also needs this: https://github.com/dotnet/runtime/blob/dbe6447aa29b14150b7c6dd43072cc75f0cdf013/tools-local/tasks/mobile.tasks/WasmAppBuilder/PInvokeTableGenerator.cs#L72